daemon/main.go

```diff
@@ -360,6 +360,7 @@ func init() {
 	flags.StringVar(&cgroupRoot,
 		"cgroup-root", "", "Path to Cgroup2 filesystem")
 	flags.Bool(option.BPFCompileDebugName, false, "Enable debugging of the BPF compilation process")
+	flags.Bool(option.SockopsEnableName, false, "Enable sockops when kernel supported")
```

Comment:
Define the default value via `pkg/defaults`

Reply:
```
--- a/pkg/defaults/defaults.go
+++ b/pkg/defaults/defaults.go
@@ -81,6 +81,9 @@ const (
        // DefaultCgroupRoot is the default path where cilium cgroup2 should be mounted
        DefaultCgroupRoot = "/mnt/cilium_cgroup2/"
 
+       // SockopsEnable controsl whether sockmap should be used
+       SockopsEnable = false
+
        // DefaultMapRootFallback is the path which is used when /sys/fs/bpf has
        // a mount, but with the other filesystem than BPFFS.
        DefaultMapRootFallback = "/run/cilium/bpffs"
```

Reply:
seems convention of starting defaults with 'Default' is hit or miss didn't add it here.